kit/Delta.hpp

```diff
@@ -467,7 +433,7 @@ class DeltaGenerator {
             assert(wasInUse && "Error: delta was already un-used by another thread");
         }
 
-        TileLocation _loc;
+        TileDesc _loc;
     private:
         std::atomic<bool> _inUse; // thread debugging check.
         TileWireId _wid;
```

Comment:
If we go to the expense and memory weight of having a full TileDesc - then we should at least try to re-use the WireId in there surely ? Then again _id - is included into the == comparison operator we use below for DeltaCompare - so probably we do have to waste that.